src/targets/gpu/kernels/include/migraphx/kernels/type_traits.hpp

```diff
@@ -246,6 +248,9 @@ constexpr T numeric_max()
         return __FLT_MAX__;
     else if constexpr(is_same<T, migraphx::half>{})
         return __FLT16_MAX__;
+    // TODO: Do it generically for all fp8 types
+    else if constexpr(is_same<T, migraphx::fp8::fp8e4m3fnuz>{})
+        return migraphx::fp8::numeric_limits<T>::max();
```

Comment:
Just add a float8 overload for `numeric_max` and `numeric_lowest` in the float8 header.

Reply:
Done.